app/src/androidTest/java/ch/epfl/cs311/wanderwave/model/DataClassesTest.kt

```diff
@@ -168,7 +169,9 @@ class DataClassesTest {
             "isPublic" to true,
             "profilePictureUri" to "https://example.com/image.jpg",
             "chosenSongs" to listOf<DocumentReference>(),
-            "topSongs" to listOf<DocumentReference>())
+            "topSongs" to listOf<DocumentReference>(),
+            "bannedSongs" to listOf<DocumentReference>(),
+            "likedSongs" to listOf<DocumentReference>())
```

Comment:
This still references the likedSongs, which was removed. Locally this test fails, I'm not sure how it passed in the CI.